tests/app/va/va_profile/test_va_profile_client_for_profile_v3.py

```diff
@@ -209,6 +294,52 @@ def test_ut_get_telephone_raises_NoContactInfoException_if_no_mobile_telephones_
         with pytest.raises(NoContactInfoException):
             mock_va_profile_client.get_telephone_with_permission(recipient_identifier, sample_notification())
 
+    def test_ut_get_telephone_raises_NoContactInfoException_if_number_classified_as_not_mobile(
```

Comment:
```suggestion
    def test_get_telephone_raises_NoContactInfoException_if_number_classified_as_not_mobile(
```
We got rid of the test type prefix.